backend/mora/service/employee.py

```diff
@@ -13,6 +13,9 @@
 
 This section describes how to interact with employees.
 
+For more information regarding reading relations involving employees, refer to
+:http:get:`/service/(any:type)/(uuid:id)/details/`
```

Comment:
Oooh, the cross-references actually work now! I seem to remember having issues with these…